src/Components/FashCard/FlashCard.tsx

```diff
@@ -19,53 +19,58 @@ const FlashCard = ({ QuestionList }: any) => {
     );
 
     return (
-        <main className="card">
-            <div className="header">
-                <div className="header-timer">
-                    <Timer />
-                </div>
+        <main>
+            <Card>
+                <div className="FlashCardHeader">
+                    <Card.Header className="cardHeader">
+                        <div className="FlashCardHeader-timer">
+                            <Timer />
+                        </div>
+
+                        <CardGroup className="FlashCardHeader-buttons">
+                            <Button
+                                type="button"
+                                variant="secondary"
+                                className="FlashCardHeader-button"
+                                onClick={() => {
+                                    QuestionList(true);
+                                }}
+                            >
+                                Question List
+                            </Button>
 
-                <div className="header-Question-list/filter">
-                    <button
-                        type="button"
-                        className=" btn btn-info"
-                        onClick={() => {
-                            QuestionList(true);
-                        }}
-                    >
-                        Question List
-                    </button>
+                            <Button type="button" variant="secondary">
+                                Submit
+                            </Button>
```

Comment:
Button text says it's a submit button...but the button type is button? Shouldn't it be type `Submit`?